### PR TITLE
add bearer token support

### PR DESCRIPTION
**Pull request by guoygang**

### CODE REVIEW

**Comment by javeme** [comment]
```java
Results :

Failed tests:   testPerfUtilPerf(com.baidu.hugegraph.unit.perf.PerfUtilTest): -110594225494

Tests run: 335, Failures: 1, Errors: 0, Skipped: 0
```
https://github.com/hugegraph/hugegraph-common/blob/4ec51b19aca1430a9bc1623fd0b41b8d407c072f/src/main/java/com/baidu/hugegraph/perf/NormalStopwatch.java#L271

File: src/main/java/com/baidu/hugegraph/perf/NormalStopwatch.java [hunk 1]
```diff
@@ -268,7 +268,9 @@ protected static void initEachWastedLost() {
             test.run();
             long end = PerfUtil.now();
             long cost = end - start - baseCost;
-            assert cost > 0 : cost;
+            if (cost < 0) {
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/perf/NormalStopwatch.java:271]: 0L

File: src/main/java/com/baidu/hugegraph/perf/NormalStopwatch.java [hunk 2]
```diff
@@ -268,7 +268,9 @@ protected static void initEachWastedLost() {
...
+                cost = 0;
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/perf/NormalStopwatch.java:272]: ditto

Comment (guoygang) [reply src/main/java/com/baidu/hugegraph/perf/NormalStopwatch.java]: done

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**